.github/workflows/mac.yaml

```diff
@@ -0,0 +1,21 @@
+name: macos
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: macos-13
+    steps:
+      - uses: actions/checkout@v4
+      - run: |
```

Comment:
Mind documenting these steps with [names](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsname)?